assets/dev/js/editor/document/elements/commands/copy.js

```diff
@@ -24,7 +24,10 @@ export class Copy extends CommandBase {
 
 		elementorCommon.storage.set(
 			storageKey,
-			containers.map( ( container ) => container.model.toJSON( { copyHtmlCache: true } ) )
+			containers.sort( ( first, second ) => {
+				const elements = elementor.getPreviewView().$el.find( '.elementor-element' );
```

Comment:
Please move above the callback, it shouldn't be queried for each element sort callback

Reply:
Done.